src/backend/ProcessLauncher.cpp

```diff
@@ -152,7 +152,7 @@ void ProcessLauncher::onLaunchRequested(const model::GameFile* q_gamefile)
     workdir = helpers::abs_workdir(workdir, game.relative_basedir, default_workdir);
 
 
-    beforeRun();
+    beforeRun(gamefile.fileinfo.absolutePath());
```

Comment:
I just noticed, this should be `absoluteFilePath`, as `absolutePath` is for returning the directory (like above with workdir).

Reply:
Yeah, I just hit this myself! Don't know how I missed it....